skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -28,23 +28,7 @@ if(MSVC)
   target_include_directories(CommonLibSSE PUBLIC "${lib_root}/include" "${third_party}")
   target_compile_options(CommonLibSSE PUBLIC "/FI\"ForceInclude.h\"" "/FI\"SKSE/Logger.h\"")
   target_link_libraries(CommonLibSSE PUBLIC Version)
-
-  file(GLOB_RECURSE common_src "${third_party}/common/*")
-  add_library(common STATIC ${common_src})
-  target_compile_options(common PRIVATE "/FI\"IPrefix.h\"")
-  target_include_directories(common PRIVATE "${third_party}/common" "${third_party}")
-
-  file(GLOB_RECURSE skse64_common_src "${third_party}/skse64_common/*")
-  add_library(skse64_common STATIC ${skse64_common_src})
-  target_compile_options(skse64_common PRIVATE "/FI\"IPrefix.h\"")
-  target_include_directories(skse64_common PRIVATE "${third_party}/common" "${third_party}/skse64_common" "${third_party}")
-
-  file(GLOB_RECURSE skse_src "${third_party}/skse64/*")
-  add_library(skse64 STATIC ${skse_src})
-  target_compile_options(skse64 PRIVATE "/FI\"IPrefix.h\"")
-  target_include_directories(skse64 PUBLIC "${third_party}/common" "${third_party}/skse64" "${third_party}")
-  target_compile_definitions(skse64 PRIVATE RUNTIME=1 RUNTIME_VERSION=0x01050610)
-  target_link_libraries(skse64 PUBLIC common skse64_common)
+  list(APPEND VCPKG_DEPENDENT CommonLibSSE) # for xbyak
```

Comment:
What about submodule? Do we still need it?

Reply:
This line has been added to keep CommonLibSSE target buildable: it requires xbyak that is shipped with SKSE. SKSE is now a vcpkg dependency so we need to add CommonLibSSE to `VCPKG_DEPENDENT`.